python/RDS_SNAPSHOT_ENCRYPTED/RDS_SNAPSHOT_ENCRYPTED.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+"""""
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  RDS_SNAPSHOT_ENCRYPTED
+
+Description:
+  Check whether AWS RDS Snapshot is encrypted.
```

Comment:
Check whether the DB Snapshots for your Amazon RDS DB Instances are encrypted. The rule is NON_COMPLIANT if the DB Snapshot is not encrypted.